app/models/manageiq/providers/autosde/storage_manager/ems_refresh_workflow.rb

```diff
@@ -80,8 +80,20 @@ def refresh
 
       queue_signal(:poll_refresh)
     end
+
   end
 
+  def queue_refresh_task(target, id = nil)
+    task_options = {
+      :action => "EmsRefresh completed successfully ",
+      :userid => "system"
+    }
+
+    EmsRefresh.queue_refresh_task(target, id, task_options)
+  end
+
+
+
```

Comment:
Minor style, but can you trim these multiple blank lines to a single blank line?